source/Engine.cpp

```diff
@@ -837,8 +837,17 @@ void Engine::Step(bool isActive)
 		targetSwizzle = target->GetSwizzle();
 		info.SetString("mission target", target->GetPersonality().IsTarget() ? "(mission target)" : "");
 
+		// Only update the "active" state shown for the target if it is
+		// in the current system and targetable, or owned by the player.
 		int targetType = RadarType(*target, step);
-		info.SetOutlineColor(GetTargetOutlineColor(targetType));
+		const bool blinking = targetType == Radar::BLINK;
+		if(!blinking || ((target->GetSystem() == player.GetSystem() && target->IsTargetable()) || target->IsYours()))
```

Comment:
```suggestion
		if(!blinking && (((target->GetSystem() == player.GetSystem() && target->IsTargetable()) || target->IsYours())))
```

Reply:
I don't think the extra layer of brackets is appropriate.

```suggestion
		if(!blinking && ((target->GetSystem() == player.GetSystem() && target->IsTargetable()) || target->IsYours()))
```